Mad.Api/Mad.Api/Controllers/WeatherForecastController.cs

```diff
@@ -48,5 +48,20 @@ public IActionResult DeleteCurrentWeatherTest()
             return Ok();
         }
 
+        [HttpPost("post-current")]
+
+        public string NotificationToCivilians()
+        {
+            var NowWeather = new WeatherForecast();
```

Comment:
```suggestion
            var nowWeather = new WeatherForecast();
```
local variebles you have to write by lowercase